Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in sandthorn.gemspec
 gemspec
+
+gem "sandthorn_driver_sequel", :git => "https://github.com/Sandthorn/sandthorn_driver_sequel.git", :branch => "remove_attribute_deltas_from_event"
```

Comment:
These two changes need to happen in lock-step, right? Breaking changes?

Reply:
Actually its only the tests that has changed in the `sandthorn_driver_sequel` gem.

What makes this PR a breaking change is that the internal structure of the event has changed.